src/web/lib/components/AppHeader/index.js

```diff
@@ -1,26 +1,23 @@
 import React from "react";
 import ReactSVG from "react-svg";
 
+import { version } from "../../../../../package.json";
 import { surveyUrl } from "../../../../lib/constants";
 import Metrics from "../../../../lib/metrics";
 
 import iconFeedback from "./feedback.svg";
 import "./index.scss";
 
-export const AppHeader = ({ hasExtension, appVersion = null }) => {
-  let survey = `${surveyUrl}?ref=app`;
-  if (appVersion) {
-    survey += `${survey}&ver=${appVersion}`;
-  }
+export const AppHeader = ({ hasExtension }) => {
+  const survey = `${surveyUrl}?ref=app?ver=${version}`;
```

Comment:
The second `?` here should be a `&` - otherwise you we get an URL with `?ref=app?ver=0.0.35`